.github/workflows/docker-image.yml

```diff
@@ -63,6 +69,14 @@ jobs:
             --target $TARGET \
             --tag $image_name .
 
+          docker buildx build \
+            --cache-from "type=local,src=/tmp/.buildx-cache" \
+            --cache-to "type=local,dest=/tmp/.buildx-cache" \
+            --output "type=image,push=true" \
+            --platform="${PLATFORM}" \
+            --target $TARGET \
+            --tag quay.io/$image_name .
```

Comment:
Unfortunately, we cannot simply build the image once and re-tag because `docker buildx build` pushes manifests to the registry as part of the process

Reply:
We do do this when possible though -- when building the windows image